### ISSUE TITLE
field serde specification does not work?

### ISSUE THREAD

**Issue by psarka** [Fatal1ty/mashumaro#53]
Following the README, I'm trying to add a custom serde for a numpy array:

```python
from dataclasses import dataclass
from dataclasses import field
import pickle

from mashumaro import DataClassMessagePackMixin
import numpy as np

@dataclass
class Numpy(DataClassMessagePackMixin):
    data: np.ndarray = field(metadata={'serialize': pickle.dumps, 'deserialize': pickle.loads})
````

It does not seem to work, as mashumaru complains:

```
UnserializableField: Field "data" of type numpy.ndarray in __main__.Numpy is not serializable
```

Python 3.8.10, mashumaro 2.6.2 

P.S. (also consider adding `mashumaro.__version__` property, currently I could not figure out how to check the mashumaro version without trying to pip install it again)

**Comment by Fatal1ty** [comment]
Hi. I fixed it in 2.6.3.

> P.S. (also consider adding mashumaro.__version__ property, currently I could not figure out how to check the mashumaro version without trying to pip install it again)

There is no need to add `__version__` because there are common ways to check a package version. On Python 3.8 you can use [importlib.metadata](https://docs.python.org/3/library/importlib.metadata.html) package. For older versions there is [pkg_resources](https://setuptools.readthedocs.io/en/latest/pkg_resources.html#getting-or-creating-distributions).

**Comment by psarka** [comment]
> Hi. I fixed it in 2.6.3.

Wow, that was fast! Thank you!

> There is no need to add `__version__` because there are common ways to check a package version. On Python 3.8 you can use [importlib.metadata](https://docs.python.org/3/library/importlib.metadata.html) package.

Thanks, was not aware of it! :+1:
